guides/v2.1/frontend-dev-guide/theme-best-practice.md

```diff
@@ -18,7 +18,7 @@ These approaches give higher chances to avoid conflicts and issues with your the
 Some other customizations that can be performed using layout instructions are the following:
  
    * change the position of a block or container using `<move>` 
-   * remove a block or container using the `remove` or `display` attribute of the `<referenceBlock>/<referenceContainer>` instruction
+   * remove a block or container using the `remove` by value `true/false` or `display` by value `true/false` attribute of the `<referenceBlock>/<referenceContainer>` instruction
```

Comment:
Hi @abrarpathan19. In this case I would clarify:

"remove a block or container by assigning `true` value to `remove` attribute or by assigning `false` value to `display` attribute".

since we are talking about removing in both cases.

Thank you

Reply:
@rogyar I have applied your suggestion please review. 
Thanks